### PR TITLE
added cera clean migration

### PR DESCRIPTION
**Pull request by Vegemash**
I removed the parts that strip out data. These seem dangerous to have in a migration.

### CODE REVIEW

File: protected/migrations/m170821_053800_cera_clean.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?php
+
+class m170821_053800_cera_clean extends CDbMigration
+{
+//	public function up()
```
Comment (marshl) [inline protected/migrations/m170821_053800_cera_clean.php]: These comments can be removed

File: protected/migrations/m170821_053800_cera_clean.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<?php
+
+class m170821_053800_cera_clean extends CDbMigration
```
Comment (marshl) [inline protected/migrations/m170821_053800_cera_clean.php]: This should inherit from OEMigration

File: protected/migrations/m170821_053800_cera_clean.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            CALL make_cera;
+            DROP PROCEDURE IF EXISTS make_cera;
+            //
+	    ')->execute();
+
+	}
+
+	public function safeDown()
+	{
+
```
Comment (marshl) [inline protected/migrations/m170821_053800_cera_clean.php]: Um, should something be done here?

File: protected/migrations/m170821_053800_cera_clean.php [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            
+              INSERT INTO openeyes.import_source
+              (name, created_user_id, last_modified_user_id, created_date, last_modified_date)
+              VALUES  (\'Fivium Australia\', 1, 1, NOW(), NOW());
+              SET import_source_id = LAST_INSERT_ID();
+              COMMIT;
+            
+              INSERT INTO openeyes.contact (nick_name, primary_phone, title, first_name, last_name, qualifications, last_modified_user_id, last_modified_date, created_user_id, created_date, contact_label_id, maiden_name)
+              VALUES (\'NULL\', \'NULL\', null, \'\', \'\', null, 1, NOW(), 1, NOW(), null, null);
+              SET contact_id = LAST_INSERT_ID();
```
Comment (marshl) [inline protected/migrations/m170821_053800_cera_clean.php]: You could actually do this in yii code by using
```
Yii::app()->db->getLastInsertId();
```

File: protected/migrations/m170821_053800_cera_clean.php (hunk 3 above)
Comment (marshl) [inline protected/migrations/m170821_053800_cera_clean.php]: nevermind

**Review by marshl — COMMENTED** [review]
We need to have a talk about your commit messages

File: protected/migrations/m170821_053800_cera_clean.php [hunk 5]
```diff
@@ -17,40 +17,28 @@ class m170821_053800_cera_clean extends CDbMigration
...
+	    $institution->save();
+
+	    $site = new Site;
+	    $site->name = 'Center for Eye Research Australia';
+	    $site->remote_id = 'CERA';
+	    $site->short_name = 'CERA';
+	    $site->telephone = '0399298360';
+	    $site->institution_id = $institution->id;
+	    $site->contact_id = $contact->id;
+	    $site->source_id = $source->id;
```
Comment (marshl) [inline protected/migrations/m170821_053800_cera_clean.php]: Should the $site be saved here?

File: protected/migrations/m170821_053800_cera_clean.php [hunk 6]
```diff
@@ -17,40 +17,28 @@ class m170821_053800_cera_clean extends CDbMigration
...
-              (name, remote_id, short_name, location_code, fax, telephone, last_modified_user_id, last_modified_date, created_user_id, created_date, institution_id, location, contact_id, replyto_contact_id, source_id, active)
-              VALUES (\'Center for Eye Research Australia\', \'CERA\', \'CERA\', \'\', \'\', \'0399298360\', 1, NOW(), 1, NOW(), institution_id, \'\', contact_id, null, import_source_id, 1);
-              COMMIT;
-            END;
-            CALL make_cera;
-            DROP PROCEDURE IF EXISTS make_cera;
-            //
-	    ')->execute();
+	    $source = new ImportSource;
+	    $source->name = 'Fivium Australia';
```
Comment (apigram) [inline protected/migrations/m170821_053800_cera_clean.php]: I was under the impression that the import source is actually to identify the PAS?

Comment (Vegemash) [inline protected/migrations/m170821_053800_cera_clean.php]: Which PAS should I be indicating this information was imported from? Or should I just create a new one from scratch to maintain a perceived meaning not actually inherent in the data model? Surely the point of this table is to show where the data came from, in this case it came from us.

Comment (apigram) [inline protected/migrations/m170821_053800_cera_clean.php]: Could just label it 'Centre for Eye Research Melbourne'.

**Review by marshl — APPROVED** [review]

**Pull request merged**